core/common/junit/src/main/java/org/eclipse/edc/junit/annotations/ApiTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (c) 2022 Microsoft Corporation
+ *  Copyright (c) 2020, 2022 Microsoft Corporation
```

Comment:
shouldn't be a dash used to separate two not contiguous years?

Reply:
Yeah right my template gone wild :)